ansible/group_vars/hub.yml

```diff
@@ -34,7 +34,10 @@ hub_containers:
     name: hub-fe
   - image: mojdigitalstudio/digital-hub-node
     name: hub-node
-
+  - image: matomo:3-apache
+    name: hub-matomo
+  - image: mojdigitalstudio/digital-hub-db
```

Comment:
Is this the same image as mariadb?

Reply:
It's the same image, but I was aiming to keep the databases separate as per current prod environment.

Reply:
OK